assets/css/style.scss

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
FYI, this is needed since we're know using a custom head that references the file. Learn more here: https://github.com/jekyll/minima#customizing-templates